Update from auto-update/renku-graph-2.22.2 (#2801)
SwissDataScienceCenter/renku: CHANGELOG.rst

0.19.2
------

> This minor bugfix release comes with internal changes to the Knowledge Graph and a other bug fixes.

Reviewer: ```suggestion
This minor bugfix release comes with internal changes to the Knowledge Graph and a bug fix on the core service.
```

> This minor bugfix release comes with internal changes to the Knowledge Graph and a bug fix on the core service.

Reviewer 2: ```suggestion
This minor bugfix release comes with internal changes to the Knowledge Graph and fixes a glitch on the core service.
```

Bug fixes
~~~~~~~~~~

> * **Knowledge Graph**: Lineage API to read data from the new dataset in the Triples Store
> * **Knowledge Graph**: migration fixing creation dates on Plans in the Triples Store
> * **Renku core service**: a bug causing Plan creation date to be after Activity start date

Reviewer 2: Should we always start these sentences with a verb? I'm not sure we have a fixed rule for that, but that's what we tired to do lately. Feel free to reword any of the suggested sentences

```suggestion
* **Knowledge Graph**: read lineage data from the new dataset in the Triples Store
* **Renku core service**: prevent Plans creation dates from preceding their corresponding Activities
* **Knowledge Graph**: fix wrong Plans creation dates through migration
```
